src/Amalgam/importexport/FileSupportJSON.cpp

```diff
@@ -333,7 +333,7 @@ std::string EvaluableNodeJSONTranslation::EvaluableNodeToJson(EvaluableNode *cod
 	if(EvaluableNodeToJsonStringRecurse(code, json_str, sort_keys))
 		return json_str;
 	else
-		return "";
+		return "nas";
```

Comment:
update the comment so it says 'otherwise return nas'

Reply:
Done.